pos_invoice_journal/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I don't care about this as I used https://github.com/acsone/bobtemplates.odoo/blob/master/bobtemplates/odoo/addon/%2Baddon.name%2B/static/description/icon.svg.oca cc @sbidoul